README.md

```diff
@@ -11,9 +11,9 @@
 
 ## Packages
 
-- [toast-ui.image-editor](https://github.com/nhn/tui.image-editor/tree/main/apps/image-editor) - Plain JavaScript component implemented by [NHN](https://github.com/nhn).
-- [toast-ui.vue-image-editor](https://github.com/nhn/tui.image-editor/tree/main/apps/vue-image-editor) - **Vue** wrapper component is powered by [NHN](https://github.com/nhn).
-- [toast-ui.react-image-editor](https://github.com/nhn/tui.image-editor/tree/main/apps/react-image-editor) - **React** wrapper component is powered by [NHN](https://github.com/nhn).
+- [toast-ui.image-editor](https://github.com/nhn/tui.image-editor) - Plain JavaScript component implemented by [NHN](https://github.com/nhn).
+- [toast-ui.vue-image-editor](https://github.com/nhn/toast-ui.vue-image-editor) - **Vue** wrapper component is powered by [NHN](https://github.com/nhn).
+- [toast-ui.react-image-editor](https://github.com/nhn/toast-ui.react-image-editor) - **React** wrapper component is powered by [NHN](https://github.com/nhn).
```

Comment:
Thank you for your contribution.
`toast-ui.vue-image-editor`, `toast-ui.react-image-editor` repository is scheduled to be deprecated.
I'd appreciate it if you could change it to the following links.
1. https://github.com/nhn/tui.image-editor/tree/master/apps/image-editor
2. https://github.com/nhn/tui.image-editor/tree/master/apps/vue-image-editor
3. https://github.com/nhn/tui.image-editor/tree/master/apps/react-image-editor

Reply:
I reflected it. f6397fb